data/org.eclipse.birt.report.data.oda.jdbc/src/org/eclipse/birt/report/data/oda/jdbc/JDBCDriverManager.java

```diff
@@ -313,6 +317,29 @@ private Connection doConnect(String driverClass, String url, String jndiNameUrl,
 		}
 	}
 
+	/*
+	 * Remove unsupported properties from the connection properties
+     * based on the driver property information set
```

Comment:
It looks like there is some inconsistent space/tabbing here.

Reply:
Crazy, on my eclipse it looks ok:

![grafik](https://github.com/user-attachments/assets/aca34ec6-0861-4c53-9c0a-6f3235d15587)